src/Distribution/Nixpkgs/Nodejs/Cli.hs

```diff
@@ -87,12 +67,9 @@ cli = \case
 
 die' :: Text -> IO a
 die' err = putErrText err *> exitFailure
-dieWithUsage :: Text -> IO ()
-dieWithUsage err = die' (err <> "\n" <> usage)
```

Comment:
What happened to `dieWithUsage`?

Reply:
I don't think it's possible to print the optparse help text there and I didn't want to duplicate it. It might be possible to extract the usage from a `ParserInfo` or similar, I'm not sure.

Reply:
Should work if you use

```
parserUsage :: ParserPrefs -> Parser a -> String -> Doc
```